packages/neos-ui/src/manifest.js

```diff
@@ -565,8 +565,14 @@ manifest('main', {}, globalRegistry => {
                 nodes
             })
         );
-        store.dispatch(actions.CR.Nodes.focus(contextPath, fusionPath));
-        store.dispatch(actions.UI.ContentCanvas.requestScrollIntoView(true));
+
+        const currentlyFocusedNodeContextPaths =
+            selectors.CR.Nodes.focusedNodePathsSelector(store.getState());
+
+        if (currentlyFocusedNodeContextPaths.includes(contextPath)) {
+            store.dispatch(actions.CR.Nodes.focus(contextPath, fusionPath));
+            store.dispatch(actions.UI.ContentCanvas.requestScrollIntoView(true));
+        }
```

Comment:
that seems to be part of the second fix you mentioned that i dont really understand at first glance. But i will try some debugging and see if i can then make sense of those lines plus your description ;)